odata-url-conventions/4 Resource Path.md

```diff
@@ -1030,16 +1031,20 @@ $filter=LastName%20eq%20'P%26G'&$select=FirstName,LastName
 ```
 :::
 
+If the resource path is passed in the request body, it MUST be percent-encoded,
+come before any query options and be separated from them by a `?`.
```

Comment:
```suggestion
come before any query options, and be separated from them by a `?`.
```

Reply:
In other words: standard representation of a (relative) URL.

Can the resource path be a relative URL, and is it then resolved relative to the request URL?

Is only `<service-root>/$query` allowed, or can I use any URL suffixed with `/$query`?